xbmc/addons/AddonManager.cpp

```diff
@@ -1088,16 +1088,6 @@ bool CAddonMgr::IsFromOfficialRepo(const AddonPtr& addon, bool bCheckAddonPath)
          std::any_of(m_officialAddonRepos.begin(), m_officialAddonRepos.end(), comparator);
 }
 
-AddonOriginType CAddonMgr::GetAddonOriginType(const AddonPtr& addon) const
-{
-  if (addon->Origin() == ORIGIN_SYSTEM)
-    return AddonOriginType::SYSTEM;
-  else if (!addon->Origin().empty())
-    return AddonOriginType::REPOSITORY;
-  else
-    return AddonOriginType::MANUAL;
-}
-
```

Comment:
I would also leave this in. It can be used currently and will be used again once we move to zip/manual installs.